developers/weaviate/manage-data/classes.mdx

```diff
@@ -187,13 +206,47 @@ The below defines:
 
 </details>
 
+<details>
+  <summary>With nested objects</summary>
+
+Weaviate `1.22` added support for nested objects. This can be very useful for storing structured data, such as an address containing a street, city and country.
```

Comment:
Consider

Weaviate `1.22` adds support for nested objects. This can be very useful for storing structured data such as an address containing a street, city and country.